docs/ReleaseNotes.md

```diff
@@ -1,4 +1,12 @@
 
+# UADK Release v2.9 June 2025
+
+## Features:
+- Support hmac(sm3)-cbc(sm4) algorithm
+- Add a high-performance mode for the ECC algorithm
+- Remove trng algorithm
+- Update CI testing tools
```

Comment:
Support hmac(sm3)-cbc(sm4) algorithm
Add a high-performance mode for the ECC algorithm
Removed trng algorithm
Update CI testing tools